notebooks/byod.py

```diff
@@ -98,30 +98,39 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
+        sample, ext = filename.str.rsplit(".", 1)
```

Comment:
Should this return `a.cram, crai` for the crai file and `a, cram` for the cram file when the crai file is called `a.cram.crai`?

Reply:
Yes, that's intended, although I just realized that will cause issues during upload_tables(). I'll mull this over and try to think of a good fix that can handle both cases.